### ISSUE TITLE
app/config/config.yml does not contain valid YAML

### ISSUE THREAD

**Issue by xezpeleta** [elkarbackup/elkarbackup#448]
In Docker image I got the following errors when I try to run `bin/console` commands:

```
In FileLoader.php line 168:
                                                                                                   
  The file "/app/elkarbackup/app/config/config.yml" does not contain valid YAML: The reserved ind  
  icator "@" cannot start a plain scalar; you need to quote the scalar at line 80 (near "- [pushP  
  rocessor, [@BnvWebUserLoggerProcessor]]") in /app/elkarbackup/app/config/config.yml (which is b  
  eing imported from "/app/elkarbackup/app/config/config_prod.yml").                               
                                                                                                   

In YamlFileLoader.php line 663:
                                                                                                   
  The file "/app/elkarbackup/app/config/config.yml" does not contain valid YAML: The reserved ind  
  icator "@" cannot start a plain scalar; you need to quote the scalar at line 80 (near "- [pushP  
  rocessor, [@BnvWebUserLoggerProcessor]]").                                                       
                                                                                                   

In Inline.php line 355:
                                                                                                   
  The reserved indicator "@" cannot start a plain scalar; you need to quote the scalar at line 80  
   (near "- [pushProcessor, [@BnvWebUserLoggerProcessor]]").                                       
                                                                                               
```

The fix seems easy (add missing quotation marks).

**Comment by elacunza** [comment]
Xabi, Uxue is already working on the issues with Docker image ;-)

**Comment by xezpeleta** [comment]
Great!!

Good work @uanabitarte  by the way!! :clap: :clap:

**Comment by xezpeleta** [comment]
Fixed in #449
